{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "37f0b122_6a38ff3a",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 5
      },
      "lineNbr": 0,
      "author": {
        "id": 1132673
      },
      "writtenOn": "2021-08-06T00:35:32Z",
      "side": 1,
      "message": "+2 although I would have been much happier if this was android::base::Result :)",
      "revId": "a622229b412133df29e4d2f1fec18728565cff92",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "6969a641_350758b6",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 5
      },
      "lineNbr": 0,
      "author": {
        "id": 1120458
      },
      "writtenOn": "2021-08-06T00:37:40Z",
      "side": 1,
      "message": "Sure, we could use it. Actually though, since Yifan has added some new usages of android::base::Result which I had to change here, I realized another problem with it. The type of the \u0027int\u0027 is ambiguous. It isn\u0027t clear if the Result contains a status_t or an errno, right? Or am I missing something?",
      "parentUuid": "37f0b122_6a38ff3a",
      "revId": "a622229b412133df29e4d2f1fec18728565cff92",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "f77c9cef_a24fa747",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 5
      },
      "lineNbr": 0,
      "author": {
        "id": 1132673
      },
      "writtenOn": "2021-08-06T00:56:37Z",
      "side": 1,
      "message": "I am suggesting to convert `status_t` to `Result\u003c\u003e`.\n\n```\nResult\u003c\u003e myfunc() {\n  status_t status \u003d function_that_returns_status_t();\n  return status \u003d\u003d OK ? {} : Error(-status) \u003c\u003c \"contextual error message\";\n}\n```\n\nIf needed, we could introduce a convenient conversion function in result.h",
      "parentUuid": "6969a641_350758b6",
      "revId": "a622229b412133df29e4d2f1fec18728565cff92",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "d8ea4d1d_205eb3e5",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 5
      },
      "lineNbr": 0,
      "author": {
        "id": 1120458
      },
      "writtenOn": "2021-08-06T16:57:13Z",
      "side": 1,
      "message": "I see, so Result\u003c\u003e always uses errno as the code?",
      "parentUuid": "f77c9cef_a24fa747",
      "revId": "a622229b412133df29e4d2f1fec18728565cff92",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}